### DISCUSSION TITLE
TensorMechanicsPlasticOrthotropic documentation

### DISCUSSION THREAD

**Discussion by mlesueur** [idaholab/moose#20824] [Q&A Modules: Solid mechanics] [answered]
Hi everyone,
I would like to model the mechanical behaviour of a material with J2 plasticity that has different values of strength in the z direction than in the x/y direction, i.e. orthotropic. From the description, it would seem that the [TensorMechanicsPlasticOrthotropic](https://github.com/idaholab/moose/blob/next/modules/tensor_mechanics/src/userobjects/TensorMechanicsPlasticOrthotropic.C) userobject is meant to do exactly this. However, there is no documentation on the object. So I would like to ask what is the model used for the implementation in order to understand the parameters to input.
Thank you in advance!

Martin

**Comment by GiudGiud** [comment, marked as answer]
@alcoae @cpritam @dschwen any ideas on this?

**Reply by alcoae** [reply]
It is based on the model proposed by this paper:

Yoon, J. W., Y. Lou, J. Yoon, and M. V. Glazoff, 2014, “Asymmetric yield function
based on the stress invariants for pressure sensitive metals,” Int. J. Plast., 56, pp.
184–202.
